Add Ctrl-i alias for Windows

lesleyrs opened the pull request:
Closes https://github.com/helix-editor/helix/issues/4874, closes https://github.com/helix-editor/helix/issues/4864

Tested on both Linux and Windows, the aliases don't seem to conflict each other.

"tab" on Linux works as both tab and C-i and "C-i" itself doesn't do anything on gnome terminal.

While on Windows tab works as tab and "C-i" works as C-i.

archseer commented:
Can you remove the unrelated tutor fixes? I think we already rejected those changes in a different PR

lesleyrs commented:
I don't think that's the case because I added them because of this yesterday https://github.com/helix-editor/helix/issues/4864#issuecomment-1333635640.

Removed it anyway.
